cli/dcoscli/marathon/main.py

```diff
@@ -340,7 +340,7 @@ def _get_resource(resource):
         raise DCOSException(
             "We currently don't support reading from the TTY. Please "
             "specify an application JSON.\n"
-            "Usage: dcos marathon app add < app_resource.json")
+            "E.g.: dcos marathon app add your-app-id < app_resource.json")
```

Comment:
Actually, this is not correct. `dcos marathon app add` doesn't take an app id. The app id is part of `app_resource.json`.